libr/bin/format/pe/pe.c

```diff
@@ -2467,6 +2467,9 @@ static void bin_pe_get_certificate(struct PE_ (r_bin_pe_obj_t) * bin) {
 	bin->cms = NULL;
 	size = bin->data_directory[PE_IMAGE_DIRECTORY_ENTRY_SECURITY].Size;
 	vaddr = bin->data_directory[PE_IMAGE_DIRECTORY_ENTRY_SECURITY].VirtualAddress;
+	if (size == 0) {
```

Comment:
Unrelated

Reply:
unfortunately it is related, because i've changed some "size" type to `size_t`.

Reply:
and later on `size - 8` is passed to `r_buf_read_at`, so it would become a very large size_t number. Actually, this check should be `size < 8`.